apps/web/app/features/stacking/direct-stacking-info/components/liquid-stacking-info-grid.layout.tsx

```diff
@@ -0,0 +1,42 @@
+import { css } from 'leather-styles/css';
+import { GridProps } from 'leather-styles/jsx';
+import { InfoGrid } from '~/components/info-grid/info-grid';
+
+interface LiquidStackingInfoGridLayoutProps extends GridProps {
+  primaryCell: React.ReactNode;
+  cells: [React.ReactNode, React.ReactNode, React.ReactNode, React.ReactNode];
+  bottomCells?: [React.ReactNode | undefined, React.ReactNode | undefined];
+}
+export function LiquidStackingInfoGridLayout({
+  primaryCell,
+  cells,
+  bottomCells,
+  ...props
+}: LiquidStackingInfoGridLayoutProps) {
```

Comment:
How can you adjust this so there are no gray empty boxes?

Reply:
not sure how, fake box?